vistrails/gui/uvcdat/diagnosticsDockWidget.py

```diff
@@ -852,7 +852,10 @@ def displayCell(self, res30, row, column, sheet="Sheet 1"):
          from packages.uvcdat_cdms.init import get_canvas, get_gm_attributes, original_gm_attributes
          from gui.uvcdat.uvcdatCommons import gmInfos
          Gtype = res30.type
-         G = V[Gtype]
+         if res30.type=="Taylor":
+             G=V["Taylordiagram"]
```

Comment:
@painter1 would know i think
but the code says:
```
         pm = projectController.plot_manager
         V=pm._plot_list["VCS"]
```
so V seems to be the list of plot type in vcs (???)
and 
```
             G=V["Taylordiagram"]
```
seems to indicate the the vcs graphic method type...